backend/src/test/java/com/praksa/auction/BiddingTest.java

```diff
@@ -0,0 +1,26 @@
+package com.praksa.auction;
+
+import com.praksa.auction.controller.BidController;
+import com.praksa.auction.dto.BiddingInfoDto;
+import com.praksa.auction.model.Bid;
+import com.praksa.auction.repository.BidRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import static org.junit.Assert.assertTrue;
+
+@SpringBootTest
+public class BiddingTest {
```

Comment:
What are you testing here, if it is method in controller then test class should have name `BidControllerTest`